providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/FlagdProviderTest.java

```diff
@@ -79,11 +80,14 @@ public static void init() {
     void path_arg_should_build_domain_socket_with_correct_path() {
         final String path = "/some/path";
 
-        ServiceBlockingStub mockStub = mock(ServiceBlockingStub.class);
+        ServiceBlockingStub mockBlockingStub = mock(ServiceBlockingStub.class);
+        ServiceStub mockStub = mock(ServiceStub.class);
         NettyChannelBuilder mockChannelBuilder = getMockChannelBuilderSocket();
 
         try (MockedStatic<ServiceGrpc> mockStaticService = mockStatic(ServiceGrpc.class)) {
-            mockStaticService.when(() -> ServiceGrpc.newBlockingStub(any(ManagedChannel.class)))
+            mockStaticService.when(() -> ServiceGrpc.newBlockingStub(any())) // why does any(ManagedChannel.class) fail?
```

Comment:
I'm not sure why `any(ManagedChannel.class)` doesn't work here, any ideas @toddbaert @Kavindu-Dodan ?

Reply:
Do you get any error ? 

May be you need to mock from higher level as `newBlockingStub()` expects `io.grpc.Channel`

So `any(Channel.class)` might work.

Reply:
Thanks @Kavindu-Dodan, that fixed it!
The following line fails when using `Channel.class` though despite also expecting `io.grpc.Channel`:
```
mockStaticService.when(() -> ServiceGrpc.newStub(any(Channel.class)))
                        .thenReturn(mockStub);
```
```
java.lang.NullPointerException: Cannot invoke "dev.openfeature.flagd.grpc.ServiceGrpc$ServiceStub.eventStream(dev.openfeature.flagd.grpc.Schema$EventStreamRequest, io.grpc.stub.StreamObserver)" because "this.serviceStub" is null
	at dev.openfeature.contrib.providers.flagd.FlagdProviderTest.path_arg_should_build_domain_socket_with_correct_path(FlagdProviderTest.java:101)
```
I'll just use `any()` for that call unless something jumps out